content/configuration/helm-chart-parameters.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Helm chart parameters
-weight: 40
```

Comment:
Please synchronize the Helm chart params as a part of this PR. Thanks!

Reply:
Yep, I mentioned that in the desc, as it requires other stuff to be merged before